### PR TITLE
Add chatbot-amd-gpu blog

### PR DESCRIPTION
**Pull request by andyluo7**

### CODE REVIEW

File: chatbot-amd-gpu.md [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+
+2\. Vicuna 13b – quant (4bit quantization)
+
+<img src="assets/chatbot-amd-gpu/12.png">
+
+- Latency: 1.4ms/layer, measured on MI210
+
+<!-- -->
+
+- Major Kernel: VecQuant4MatMulKernel (dequant + MatMul)
```
Comment (mfuntowicz) [inline chatbot-amd-gpu.md:327]: I would personally remove this part, it's in my opinion a bit to low-level and hard to grasp from the images.

File: chatbot-amd-gpu.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+than 10B, the 4-bit or 3-bit GPTQ can achieve the comparable accuracy
+with fp16.
+
+<p align="center">
+  <img src="assets/chatbot-amd-gpu/02.png" style="width: 70%; height: auto;">
+</p>
+  
+Moreover, large parameters of these models also have a severely negative
+effect on GPT latency because GPT token generation is more limited by
+DDR bandwidth (GB/s) than computation (TFLOPs or TOPs) itself. For this
```
Comment (mfuntowicz) [inline chatbot-amd-gpu.md:80]: Maybe "memory" instead of "DDR" for simplicity?

File: chatbot-amd-gpu.md [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+reason, a quantized model in the compressed format does not degrade
```
Comment (mfuntowicz) [inline chatbot-amd-gpu.md:81]: Remove "in the compressed format"?

File: chatbot-amd-gpu.md [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+token generation latency when the GPU is under a memory bound situation.
+Refer to the GPTQ quantization papers and github repo:
```
Comment (mfuntowicz) [inline chatbot-amd-gpu.md:83]: To improve redability maybe use clickable text links for:

- "the GPTQ quantization papers"
- "github repo"

File: chatbot-amd-gpu.md [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+
+**GPTQ Dequantization + FP16 Mamul kernel for AMD GPUs**
+
+The more optimized kernel implementation in
+<https://github.com/oobabooga/GPTQ-for-LLaMa/blob/57a26292ed583528d9941e79915824c5af012279/quant_cuda_kernel.cu#L891>
+
+targets at A100 GPU and not compatible with ROCM5.4.3 HIPIFY
+toolkits. Please modify the kernel codes as follows. Do the same for
+VecQuant2MatMulKernelFaster, VecQuant3MatMulKernelFaster,
+VecQuant4MatMulKernelFaster kernels.
```
Comment (mfuntowicz) [inline chatbot-amd-gpu.md:412]: Maybe we can use a Github Gist with the modified code and add a link to it to make it easier for people to replicate?

**Review by mfuntowicz — APPROVED** [review]
Overall LGTM, I let a few comments mainly to improve readability where it wouldn't hurt the content.

This is really personnal so feel free to discard if you think it should be present 🙂

**Comment by andyluo7** [comment]
@mfuntowicz .md file updated based on your suggestion. How to resolve conflicts for _blog.yml?

**Review by juliensimon — APPROVED** [review]
Looks good to me. Cool post!

**Comment by andyluo7** [comment]
@mfuntowicz @juliensimon I don't see the blog yet. When will it be published?

**Comment by osanseviero** [comment]
Also new assets don't go in this GitHub repo, only thumbnails. Assets are now added to https://huggingface.co/datasets/huggingface/documentation-images/tree/main/blog in order to keep the GH repository light

**Pull request merged**